data/tooltips.txt

```diff
@@ -98,6 +104,45 @@ tip "cloaking fuel:"
 tip "cloaking heat:"
 	`Heat generated per second while cloaked.`
 
+tip "cloaking shield:"
+	`Shields consumed per second while cloaked.`
+
+tip "cloaking shield delay:"
+	`This ship is unable to regenerate shields whilst cloaked, instead its shield regeneration will suffer a delay that will grow the longer it is cloaked`
+
+tip "cloaking hull:"
+	`Hull consumed per second while cloaked.`
+
+tip "cloaking repair delay:"
+	`This ship is unable to regenerate hull whilst cloaked, instead its hull regeneration will suffer a delay that will grow the longer it is cloaked.`
+
+tip "cloak hull protection:"
+	`Protection against weapons that do hull damage when the cloak is active. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in hull damage, while a total value of 11 only results in a 10 percent reduction.`
+
+tip "cloak shield protection:"
+	`Protection against weapons that do shield damage when the cloak is active. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`
+
+tip "cloak phasing"
+	`Weapons cannot affect this ship, and pass right trough.`
+
+tip "cloak visibility:"
```

Comment:
Let's keep this for another PR, as suggested in https://github.com/endless-sky/endless-sky/pull/7025#issuecomment-1205425758 , same for taretability, infrared traceability and radar traceability.

Reply:
fair if that helps getting this one merged its true that how those would work could use an issue I just added what was requested but doesnt help getting it trough :)